srv/wordpress/wp-content/plugins/acore-wp-plugin/src/components/AdminPanel/Settings.view.php

```diff
@@ -64,6 +66,13 @@ public function getSettingsRender() {
         ?>
 
         <form name="form-acore-settings" method="post" action="">
+
+        <div class="card p-0">

```

Comment:
you could improve the indentation here, we should import the plugin prettier to make it automatically on-save